llvm/test/CodeGen/AArch64/execute-only-section.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: llc -mtriple=aarch64 -mattr=+execute-only %s -o - | FileCheck %s
+
+; CHECK:     .section .text,"axy",@progbits,unique,0
```

Comment:
Should test more cases:

* a COMDAT text section
* a rodata section doesn't get `y`